client-misk-hibernate/src/test/kotlin/app/cash/backfila/client/misk/menustack/MenuStackTest.kt

```diff
@@ -0,0 +1,65 @@
+package app.cash.backfila.client.misk.menustack
+
+import app.cash.backfila.client.misk.ClientMiskService
+import app.cash.backfila.client.misk.ClientMiskTestingModule
+import app.cash.backfila.client.misk.DbMenu
+import app.cash.backfila.client.misk.MenuItem
+import app.cash.backfila.client.misk.MenuQuery
+import app.cash.backfila.client.misk.hibernate.SandwichParameters
+import app.cash.backfila.embedded.Backfila
+import app.cash.backfila.embedded.createDryRun
+import app.cash.backfila.embedded.createWetRun
+import com.google.inject.Module
+import java.util.concurrent.BlockingDeque
+import javax.inject.Inject
+import misk.hibernate.Query
+import misk.hibernate.Session
+import misk.hibernate.Transacter
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+@MiskTest(startService = true)
+class MenuStackTest {
+  @Suppress("unused")
+  @MiskTestModule
+  val module: Module = ClientMiskTestingModule(false)
+
+  @Inject @ClientMiskService
+  lateinit var transacter: Transacter
+
+  @Inject lateinit var queryFactory: Query.Factory
+
+  @Inject lateinit var backfila: Backfila
+
+  @MenuStack @Inject
+  lateinit var menuStack: BlockingDeque<MenuItem>
+
+  @Test
+  fun `create a stack of menus using a hibernate backfill and a parameter backfill`() {
```

Comment:
If this test is meant to be demonstrative, perhaps it should be documented? Something like this?

> 
```suggestion
  /**
   * Demonstrates backfilling menus from a Hibernate datasource into an in-memory static datasource.
   */
  @Test
  fun `create a stack of menus using a hibernate backfill and a parameter backfill`() {
```

Reply:
I updated the test a bit and the comment. Hopefully that is clearer now